src/components/Tabs/README.md

```diff
@@ -195,3 +195,23 @@ Also known as [Segmented controls](https://developer.apple.com/design/human-inte
 ![Fixed tabs on iOS](components/Tabs/ios/fixed.png)
 
 <!-- /content-for -->
+
+<!-- content-for: web -->
+
+---
+
+## Accessibility
```

Comment:
Here's an example where I go more into depth about the structure of the feature. I could go either way with relying on the linked spec or including specific detail with the component. If we go in the broader direction of linking to known issues or PRs for a specific component, the structure and keyboard detail might help, I think.

Reply:
IMO, linking to the spec is the right approach here. This says that we didn't just imagine the implementation, but that we are actually following industry best practices. Also the spec is pretty safe to rely on always being there, and not changing or becoming stale. I'm still iffy about linking to a specific issue or PR in the styleguide, as their lifecycle is far shorter, and more at risk of becoming stale.

In general I would think about these things in terms of lifecycle, where a resource with a long lifecycle should not link back to a resource with a shorter lifecycle. In terms of lifecycles, I see it kind of like this, ranging from shortest lifespan to longest:

`pull request > issue > component readme/styleguide page`

Other elements I see with a similar lifespan to READMEs would be a master component issue (hub with spokes to other short lifespan issues), and component metadata.

Happy to expand on this 1:1 if it makes sense to do so.